main/main_builders.py

```diff
@@ -63,68 +63,5 @@ def make_app_icon(target, source, env):
         g.write("};\n")
         g.write("#endif")
 
-
-def make_default_controller_mappings(target, source, env):
```

Comment:
For the reference if anyone wonders about this removal: this code was moved to `core/input/`.